### ISSUE TITLE
Request to add member to org for Vancouver group and create wiki instructions for future

### ISSUE THREAD

**Issue by gyaresu** [nodeschool/organizers#515]
Hi, folks.

I might have missed it but it might be helpful to have a section in the wiki on how to get members invited to the Nodeschool Org.

I couldn't see anything about it in the [wiki](https://github.com/nodeschool/organizers/wiki).

Anywho, I'd like to get [@chowdhurian](https://github.com/chowdhurian) added so we can make him an admin on nodeschool/vancouver

Thanks!

/cc @Qard

**Comment by riyadhalnur** [comment]
Makes sense. Definitely, something we should add to make it easier to make onboarding easier. PRs are always welcome.

Would you like to add @chowdhurian to the `chapter-organisers` team or just the Vancouver team only? If the latter only, then any existing member of the Vancouver can add someone in.

**Comment by gyaresu** [comment]
Hi @riyadhalnur, thanks for taking the time to respond.

Adding @chowdhurian to the `chapter-organisers` team would be great.

As for adding to Vancouver I have tried that and it stays greyed out as he's not "a member of this organisation".

All good things for the wiki once it's clear what the process is. I'm sure we could write it. Just got to figure out the workflow :)

![screen shot 2018-03-13 at 1 40 34 am](https://user-images.githubusercontent.com/29263/37330980-e64f885e-265f-11e8-9ba6-78307559777f.png)

Thanks again!

**Comment by riyadhalnur** [comment]
I went ahead and invited @chowdhurian to the Vancouver team and `chapter-organisers` team. I'll try to update the wiki once I get some time.

**Comment by riyadhalnur** [comment]
@nodeschoolbot add-user chowdhurian

**Comment by riyadhalnur** [comment]
I have added @chowdhurian to the `chapter-organizers` team.
